src/auction-dca.rain

```diff
@@ -146,6 +163,15 @@ scenarios:
       baseline-fn: '''constant-baseline'
       initial-io-fn: '''constant-initial-io'
       shy-epoch: 0.05
+  arbitrum2:
+    orderbook: arbitrum2
+    runs: 1
+    bindings:
+      raindex-subparser: 0xb06202aA3Fe7d85171fB7aA5f17011d17E63f382
```

Comment:
should be `0x762adD85a30A83722feF2e029087C9D110B6a7b3` i think? https://github.com/rainlanguage/rain.orderbook/actions/runs/15413757494/job/43371579897